.github/workflows/mirror.yml

```diff
@@ -0,0 +1,36 @@
+name: mirror
+
+on:
+  schedule:
+    - cron:  '0 */12 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+        fetch-depth: 0
+    - name: Read api.github.com and filter response
+      run: |
+        cd common/mirror
+        export JQ_FILTER="[.[] | {tag_name: .tag_name, assets: [.assets[] | {browser_download_url: .browser_download_url} ] } ]"
+        curl -H "Authorization: ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/SeleniumHQ/selenium/releases | jq "$JQ_FILTER" > selenium
+    - name: Commit files
+      id: git
+      run: |
+        export CHANGES=$(git status -s)
+        if [ -n "$CHANGES" ]; then
+           git config --local user.email "boni.garcia@uc3m.es"
+           git config --local user.name "Boni Garcia"
```

Comment:
Let's use the info from the GitHub bot instead.

Reply:
Done.

Reply:
You still need to add a user email and a user name. This is what I meant:
```
        git config --local user.email "github-actions[bot]@users.noreply.github.com"
        git config --local user.name "github-actions[bot]"
```

Reply:
Done.